### DISCUSSION TITLE
Software packages

### DISCUSSION THREAD

**Discussion by gipsy58** [terjeio/ioSender#206] [Ideas]
Thanks for your IOSender. Could you please have packages of the full versions of the IOSender available. 
After a long time of modifying my old lathe and building a controller with Phil's Teensy controller I am keen of having the additional modules (Turning/Threading) available for it without having to compile them?

**Comment by phil-barrett** [comment]
Is the [zip file release](https://github.com/terjeio/ioSender/releases/tag/2.0.36) not what you want? There is no installer, just unzip the downloaded file and run the .exe.

**Reply by gipsy58** [reply]
Thanks for the fast reply. My mistake. I got that and it runs perfectly. But i cannot see the turning and threading tabs. Do I need to do some configuration?

**Reply by terjeio** [reply]
You have to enable lathe mode:
![image](https://user-images.githubusercontent.com/20260062/154548640-c710704a-f097-45d6-a334-cad7471444b3.png)

For threading spindle sync has to be set up - Phil's board has to be modified for that. Note that the code has not been tested much - I have spent way too much time coding and nearly no time with my lathe for quite a while now...

**Comment by gipsy58** [comment]
Hi Terjeio

Thank you very much for your help.

________________________________
